dotcom-rendering/src/model/enhanceCards.ts

```diff
@@ -96,16 +96,16 @@ const decideAvatarUrl = (
 	return soleContributor?.bylineLargeImageUrl ?? undefined;
 };
 
-const enhanceTags = (tags: { properties: FETagType }[]): TagType[] => {
-	return tags.map((tag) => {
+const enhanceTags = (tags: FETagType[]): TagType[] => {
```

Comment:
It took me a minute to understand this because I hadn't got to the type definition change for `FETagType` yet, but I think the refactor makes sense 👍

Reply:
Thanks, I was confused as if something comes from Frontend it should not change shape. 😍